vhdbuilder/packer/install-dependencies.sh

```diff
@@ -178,6 +178,7 @@ for imageToBePulled in ${ContainerImages[*]}; do
 done
 
 VNET_CNI_VERSIONS="
+1.4.1
 1.4.0
```

Comment:
Going to keep 1.4.0 cached just to avoid any possible complications since that is what is currently in some regions. Will remove once it is completely unused.